src/pve_exporter/collector/node.py

```diff
@@ -126,3 +127,66 @@ def collect(self): # pylint: disable=missing-docstring
                     metrics[key].add_metric(label_values, metric_value)
 
         return itertools.chain(metrics.values(), info_metrics.values())
+
+class SubscriptionCollector:
+    """
+    Collects Proxmox VE subscription information (node, subscription level, status, next due date).
+    """
+
+    def __init__(self, pve):
+        self._pve = pve
+
+    def collect(self):  # pylint: disable=missing-docstring
+        info_metric = GaugeMetricFamily(
+            "pve_subscription_info",
+            "Proxmox VE subscription info (1 if present)",
+            labels=["id", "level"],
+        )
+
+        possible_statuses = ["new", "notfound", "active", "invalid", "expired", "suspended"]
+        status_metric = GaugeMetricFamily(
+            "pve_subscription_status",
+            "Proxmox VE subscription status (1 if matches status)",
+            labels=["id", "status"],
+        )
+
+        next_due_metric = GaugeMetricFamily(
+            "pve_subscription_next_due_timestamp_seconds",
+            "Subscription next due date as Unix timestamp",
+            labels=["id"],
+        )
+
+        node = None
+        for entry in self._pve.cluster.status.get():
+            if entry['type'] == 'node' and entry['local']:
+                node = entry['name']
+                break
+
+        subscription = self._pve.nodes(node).subscription.get()
+
+        level = subscription.get("level", "unknown")
+        status = subscription.get("status", "unknown")
+
+        info_metric.add_metric(
+            [f"node/{node}", node, level, status],
+            1,
+        )
+
+        for possible_status in possible_statuses:
+            value = 1 if status == possible_status else 0
+            status_metric.add_metric(
+                [f"node/{node}", node, possible_status],
+                value,
+            )
+
+        next_due_date = subscription.get("nextduedate")
+        if next_due_date:
+            timestamp = datetime.strptime(next_due_date, "%Y-%m-%d").timestamp()
+            next_due_metric.add_metric(
+                [f"node/{node}", node, level],
```

Comment:
Also remove `node` and `level` label from this line. The list of values need to correspond with the list of labels on the metric (see line 156).